src/plugins/timeline.ts

```diff
@@ -143,6 +145,11 @@ class TimelinePlugin extends BasePlugin<TimelinePluginEvents, TimelinePluginOpti
     return 2
   }
 
+    // Return how many seconds the labels are offset.
+    private defaultTimeOffset(pxPerSec: number): number {
+      return 0;
+    }
```

Comment:
This doesn't have to be a function, does it? It can be just a static value in [defaultOptions](https://github.com/katspaugh/wavesurfer.js/blob/2867c6d633bb275ed4d027a517ced0d192828652/src/plugins/timeline.ts#L37-L50).